components/crowdfunding-redesign/CollectivePrimaryColor.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { get } from 'lodash';
+
+function HexToHSL(hex) {
```

Comment:
You can use `parseToHsl` from polished (already installed)

https://polished.js.org/docs/#parsetohsl

Reply:
Nice, fixing this